examples/.joyride/src/util/editor.cljs

```diff
@@ -35,6 +35,22 @@
       (p/catch (fn [e]
                  (js/console.error e)))))
 
+;; update name, but don't break existing scripts
+(def delete-range!+ delete-range!)
+
+(defn replace-range!+
+  "Defaults to the current selection."
+  ([^js text]
+   (replace-range! text vscode/window.activeTextEditor (.-active (current-selection)) (current-selection)))
```

Comment:
Your REPL probably knows what `replace-range!` is, but it's not defined in the file, is it? 😄

Reply:
oops.  😳